lib/SQL/Translator/Parser/PostgreSQL.pm

```diff
@@ -1176,6 +1403,27 @@ sub parse {
   return 1;
 }
 
+# Apply PostgreSQL database's default values!
+# Because the default values are database specific,
+# they cannot be set in the Schema class Sequence.
+sub create_sequence {
+  my ($class, %data) = @_;
+
+  my $data_type = SQL::Translator::Schema::DataType->new(type => 'integer', size => 20);
+  $data{temporary} //= 0; # Boolean
+  $data{unlogged} //= 0; # Boolean
+  $data{data_type} //= $data_type; # q{bigint};
+  $data{increment} //= 1;
+  $data{minvalue} //= 1; # Same as NO MINVALUE
+  $data{maxvalue} //= 0; # Same as NO MAXVALUE
+  $data{start} //= 1;
+  $data{cache} //= 1;
+  $data{cycle} //= 0; # Boolean, Same as NO CYCLE
+  $data{owner} //= q{NONE};
```

Comment:
you set up these defaults twice, could you make them a shared hash which you conditionally load into the `$data` via `for my $key (keys %seq_defaults) { $data{$key} //= $seq_defaults{$key} }`?